pysat/instruments/pysat_testing.py

```diff
@@ -55,49 +60,60 @@ def init(self):
     """
 
     self.new_thing = True
+    logger.info(mm_test.ackn_str)
+    self.acknowledgements = mm_test.ackn_str
+    self.references = mm_test.refs
 
     # work on file index if keyword present
-    if self.kwargs['file_date_range'] is not None:
+    if self.kwargs['load']['file_date_range'] is not None:
         # set list files routine to desired date range
         # attach to the instrument object
-        fdr = self.kwargs['file_date_range']
-        self._list_rtn = functools.partial(list_files, file_date_range=fdr)
+        fdr = self.kwargs['load']['file_date_range']
+        self._list_files_rtn = functools.partial(list_files,
+                                                 file_date_range=fdr)
         self.files.refresh()
 
     # mess with file dates if kwarg option present
-    if self.kwargs['mangle_file_dates']:
+    if self.kwargs['load']['mangle_file_dates']:
         self.files.files.index = \
-            self.files.files.index + pds.DateOffset(minutes=5)
+            self.files.files.index + dt.timedelta(minutes=5)
+    return
 
 
-def default(self):
-    """Default customization function.
+def clean(self):
+    """Cleaning function
+    """
 
-    This routine is automatically applied to the Instrument object
-    on every load by the pysat nanokernel (first in queue).
+    return
 
-    Parameters
-    ----------
-    self : pysat.Instrument
-        This object
+
+# Optional method
+def preprocess(self):
+    """Customization method that performs standard preprocessing.
+
+    This routine is automatically applied to the Instrument object
+    on every load by the pysat nanokernel (first in queue). Object
+    modified in place.
 
     """
 
-    pass
+    return
 
 
-def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
+def load(fnames, tag=None, inst_id=None, sim_multi_file_right=False,
          sim_multi_file_left=False, root_date=None, file_date_range=None,
-         malformed_index=False, mangle_file_dates=False):
+         malformed_index=False, mangle_file_dates=False, num_samples=None,
+         multi_file_day=False):
     """ Loads the test files
 
     Parameters
     ----------
     fnames : list
         List of filenames
     tag : str or NoneType
-        Instrument tag (accepts '')
-    sat_id : str or NoneType
+        Instrument tag (accepts '' or a string to change the behaviour of
+        dummy1 for constellation testing)
+    inst_id : str or NoneType
         Instrument satellite ID (accepts '' or a number (i.e., '10'), which
         specifies the number of data points to include in the test instrument)
```

Comment:
I thought this wasn't supported anymore.

Reply:
Looking at the code, it's been deprecated but will still work in this branch

Reply:
Do we want to remove the functionality here and add the deprecation in 2.3, or update the deprecation documentation/warnings here?

Reply:
To be consistent with the other warnings I think the deprecation should be moved to 2.3 and support removed in 3.0. 
I put it in wrong originally.